src/main/java/homework_4/custom_file_reader/Main.java

```diff
@@ -0,0 +1,10 @@
+package homework_4.custom_file_reader;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class Main {
+    public static void main(String[] args) {
+        CustomFileReader.run();
```

Comment:
Looks strange 
![image](https://user-images.githubusercontent.com/58872509/134134035-989d1cbe-2720-4d27-9696-12d2cb228c23.png)

Reply:
How do I get this error? 
remind, i don't use windows